README.md

```diff
@@ -2,6 +2,16 @@
 
 This ``insightsoftwareconsortium-org`` branch of this repository store the static content served at the root of http://insightsoftwareconsortium.org.
 
+[![Netlify Status](https://api.netlify.com/api/v1/badges/eb97a4fa-5a24-41f7-995c-affeb811763c/deploy-status)](https://app.netlify.com/sites/insightsoftwareconsortium/deploys)
+
+Kitware hosted:
```

Comment:
Is https://insightsoftwareconsortium.org associated with https://insightsoftwareconsortium.netlify.com using DNS record ? Or are these two different webserver ?

Reply:
These are different servers currently. It looks like the Kitware server is not updating -- I will contact sys-admin.